### ISSUE TITLE
Mycoplasma infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6654]
**Preferred term label:**
Mycoplasma infection

**Synonyms**
Mycoplasma

**Definition (free text, please give PubMed ID)**
An infection caused by mycoplasma. 
PMID: 30725612

**Parent term (use hpo.jax.org/app)**
Unusual infections

**Comment by pnrobinson** [comment]
This is a disease (category) and not a phenotypic feature. Please make a term request at https://github.com/monarch-initiative/mondo
